### PR TITLE
183882576 leaders daemon for pythnet

### PR DESCRIPTION
**Pull request by maciejkocylapc**
#### What's this PR do?
- add pythnet leader daemon

#### How should this be manually tested?
- run `rails r daemons/leader_stats_pythnet_update_daemon.rb`
- check leaders component on the homepage

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183882576)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?

### CODE REVIEW

File: config/deploy.rb [hunk 1]
```diff
@@ -124,6 +124,7 @@
           execute :systemctl, '--user', :start, :front_stats_update
           execute :systemctl, '--user', :start, :leader_stats_mainnet_update
           execute :systemctl, '--user', :start, :leader_stats_testnet_update
+          # execute :systemctl, '--user', :start, :leader_stats_pythnet_update
```
Comment (kamtarnowski) [inline config/deploy.rb:127]: Should be uncommented.

File: config/deploy.rb [hunk 2]
```diff
@@ -145,6 +146,7 @@
           execute :systemctl, '--user', :stop, :front_stats_update
           execute :systemctl, '--user', :stop, :leader_stats_mainnet_update
           execute :systemctl, '--user', :stop, :leader_stats_testnet_update
+          # execute :systemctl, '--user', :stop, :leader_stats_pythnet_update
```
Comment (kamtarnowski) [inline config/deploy.rb:149]: Should be uncommented.

File: config/deploy.rb (hunk 1 above)
Comment (kbiala) [reply config/deploy.rb:127]: No difference at this point. We're not going to use this task until the code is production.

File: config/deploy.rb (hunk 2 above)
Comment (kbiala) [reply config/deploy.rb:149]: Same here

**Review by kbiala — APPROVED** [review]

**Pull request merged**